src/back/jobs/BadgeAirdrop.ts

```diff
@@ -33,3 +33,8 @@ async function giveAndRevokeLandOwnerBadges() {
     await BadgesService.giveAndRevokeLandOwnerBadges()
   }
 }
+
+export async function giveTopVoterBadges() {
```

Comment:
Do we want this to run also in testing env or only in prod?

Reply:
Good question. I think we do want it to run in testing env, since I expect for us to be the top voters on our space.